### PR TITLE
Docs: Update "What's Next?" section in PyPI installation instructions

### PR DESCRIPTION
**Pull request by EdjeElectronics**
This PR changes the What's Next? section in the PyPI installation instructions to better entice users to try out tutorials and examples. I used direct URL links to the tutorials, because I'm not sure how to link to examples on the Tutorials page.

### Details:
 - Rewrite What's Next? section in `installing-openvino-pip.md`

### Tickets:
 -NA

### CODE REVIEW

File: docs/install_guides/installing-openvino-pip.md [hunk 1]
```diff
@@ -73,23 +73,28 @@ python -c "from openvino.runtime import Core"
 
 If installation was successful, you will not see any error messages (no console output).
 
-## Installing OpenVINO Development Tools
-
-OpenVINO Development Tools include Model Optimizer, Benchmark Tool, Accuracy Checker, Post-Training Optimization Tool and Open Model Zoo tools including Model Downloader. If you want to install OpenVINO Development Tools, OpenVINO Runtime will also be installed as a dependency, so you don't need to install OpenVINO Runtime separately.
+Congratulations! You finished installing OpenVINO Runtime. Now you can start exploring OpenVINO's functionality through Jupyter Notebooks and sample applications. See the [What's Next?](#whats-next) section to learn more!
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-pip.md:76]: Again, the `[What's Next?](#whats-next)` will not work. We need to change it to the `<a>` tags.

File: docs/install_guides/installing-openvino-pip.md [hunk 2]
```diff
@@ -73,23 +73,28 @@ python -c "from openvino.runtime import Core"
...
-* To convert models for use with OpenVINO, see [Model Optimizer Developer Guide](../MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md).
-* See pre-trained deep learning models in our [Open Model Zoo](../model_zoo.md).
-* Try out OpenVINO via [OpenVINO Notebooks](https://docs.openvino.ai/latest/notebooks/notebooks.html).
-* To write your own OpenVINO™ applications, see [OpenVINO Runtime User Guide](../OV_Runtime_UG/openvino_intro.md).
-* See sample applications in [OpenVINO™ Toolkit Samples Overview](../OV_Runtime_UG/Samples_Overview.md).
+### Get started with Python
+Visit the [Tutorials](../tutorials.md) page for more Jupyter Notebooks to get you started with OpenVINO, such as:
+* [OpenVINO Python API Tutorial](https://docs.openvino.ai/latest/notebooks/002-openvino-api-with-output.html)
+* [Basic image classification program with Hello Image Classification](https://docs.openvino.ai/latest/notebooks/001-hello-world-with-output.html)
+* [Convert a PyTorch model and use it for image background removal](https://docs.openvino.ai/latest/notebooks/205-vision-background-removal-with-output.html)
```
Comment (kblaszczak-intel) [inline docs/install_guides/installing-openvino-pip.md:94]:
```suggestion
* [Basic image classification program with Hello Image Classification](https://docs.openvino.ai/2022.2/notebooks/001-hello-world-with-output.html)
* [Convert a PyTorch model and use it for image background removal](https://docs.openvino.ai/2022.2/notebooks/205-vision-background-removal-with-output.html)
```
At this stage using direct links may actually be the right way to do it. I would just use the 2022.2 address to make it more future-proof.

Comment (kblaszczak-intel) [inline docs/install_guides/installing-openvino-pip.md:94]:
```suggestion
* [OpenVINO Python API Tutorial](https://docs.openvino.ai/2022.2/notebooks/002-openvino-api-with-output.html)
* [Basic image classification program with Hello Image Classification](https://docs.openvino.ai/2022.2/notebooks/001-hello-world-with-output.html)
* [Convert a PyTorch model and use it for image background removal](https://docs.openvino.ai/2022.2/notebooks/205-vision-background-removal-with-output.html)
```
At this stage using direct links may actually be the right way to do it. I would just use the 2022.2 address to make it more future-proof.

File: docs/install_guides/installing-openvino-pip.md [hunk 3]
```diff
@@ -73,23 +73,28 @@ python -c "from openvino.runtime import Core"
...
 
+See the [Install OpenVINO Development Tools](installing-model-dev-tools.md) page for step-by-step installation instructions.
 
 ## What's Next?
+Now that you've installed OpenVINO Runtime, you're ready to run your own machine learning applications! Learn more about how to integrate a model in OpenVINO applications by trying out the following tutorials.
+
+<img src="https://user-images.githubusercontent.com/15709723/127752390-f6aa371f-31b5-4846-84b9-18dd4f662406.gif" width=400>
 
-Now you may continue with the following tasks:
+Try the [Python Quick Start Example](https://docs.openvino.ai/latest/notebooks/201-vision-monodepth-with-output.html) to estimate depth in a scene using an OpenVINO monodepth model in a Jupyter Notebook inside your web browser.
```
Comment (kblaszczak-intel) [inline docs/install_guides/installing-openvino-pip.md:88]:
```suggestion
Try the [Python Quick Start Example](https://docs.openvino.ai/2022.2/notebooks/201-vision-monodepth-with-output.html) to estimate depth in a scene using an OpenVINO monodepth model in a Jupyter Notebook inside your web browser.
```

**Review by kblaszczak-intel — APPROVED** [review]
Left a couple of suggestions. Appart from them, It looks better and better.

File: docs/install_guides/installing-openvino-pip.md (hunk 1 above)
Comment (EdjeElectronics) [reply docs/install_guides/installing-openvino-pip.md]: Okay, I changed the link. I changed it to use the direct hotlink, like Karol showed:

```
See the [What's Next](https://docs.openvino.ai/2022.2/openvino_docs_install_guides_installing_openvino_pip.html#what-s-next) section to learn more!
```

Let me know if I should change it to use <a> tags instead!

**Comment by EdjeElectronics** [comment]
Thanks @kblaszczak-intel and @xu-yuan1 for the feedback. I made the changes you suggested. Let me know if there's anything else!

File: docs/install_guides/installing-openvino-pip.md [hunk 4]
```diff
@@ -73,23 +73,28 @@ python -c "from openvino.runtime import Core"
 
 If installation was successful, you will not see any error messages (no console output).
 
-## Installing OpenVINO Development Tools
-
-OpenVINO Development Tools include Model Optimizer, Benchmark Tool, Accuracy Checker, Post-Training Optimization Tool and Open Model Zoo tools including Model Downloader. If you want to install OpenVINO Development Tools, OpenVINO Runtime will also be installed as a dependency, so you don't need to install OpenVINO Runtime separately.
+Congratulations! You finished installing OpenVINO Runtime. Now you can start exploring OpenVINO's functionality through Jupyter Notebooks and sample applications. See the [What's Next](https://docs.openvino.ai/2022.2/openvino_docs_install_guides_installing_openvino_pip.html#what-s-next) section to learn more!
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-pip.md:76]:
```suggestion
Congratulations! You finished installing OpenVINO Runtime. Now you can start exploring OpenVINO's functionality through Jupyter Notebooks and sample applications. See the <a href="#whats-next">What's Next</a> section to learn more!
```

File: docs/install_guides/installing-openvino-pip.md (hunk 1 above)
Comment (xu-yuan1) [reply docs/install_guides/installing-openvino-pip.md]: No that's not what I meant. It should be similar to this comment: https://github.com/openvinotoolkit/openvino/pull/13156#issuecomment-1258936466 
I've corrected it. :)

**Review by xu-yuan1 — APPROVED** [review]

File: docs/install_guides/installing-openvino-pip.md (hunk 1 above)
Comment (EdjeElectronics) [reply docs/install_guides/installing-openvino-pip.md]: Thanks Yuan!

**Pull request merged**